tests/test_config_flag.py

```diff
@@ -10,20 +17,19 @@ def test_load_options_function():
     assert not overrides["checks"]["uncomparables.misc"]
 
 
-def test_load_fallbacks():
-    """Test load_options with a fallback path"""
-    fallbacks = load_options(None, ["tests/test_config_flag_proselintrc"])
-    assert not fallbacks["checks"]["uncomparables.misc"]
-    fallbacks = load_options(None, ["./.proselintrc"])
-    assert fallbacks["checks"]["uncomparables.misc"]
-
-
 def test_config_flag():
     """Test the --config CLI argument"""
     output = subprocess.run(["python", "-m", "proselint", "--demo"],
                             stdout=subprocess.PIPE, encoding='utf-8')
     assert "uncomparables.misc" in output.stdout
+
     output = subprocess.run(["python", "-m", "proselint", "--demo", "--config",
                              "tests/test_config_flag_proselintrc"],
                             stdout=subprocess.PIPE, encoding='utf-8')
     assert "uncomparables.misc" not in output.stdout
+
+    output = subprocess.run(["python", "-m", "proselint", "--demo", "--config",
+                             "non_existing_file"],
+                            stderr=subprocess.PIPE, encoding='utf-8')
+    assert output.returncode == 2
+    assert 'FileNotFoundError' in output.stderr
```

Comment:
> It appears that for this PR most of the coverage loss stems from proselint/command_line.py.
> To resolve the coverage issues, you need to:
> * Add testing that triggers the sys.exit(2) statement
> * Add testing for if the configuration file is not a file
>
> I will adjust the Codecov settings to set a threshold for coverage loss until then.

Reply:
Apologies, I didn't see this until just now. I'll investigate later tonight.

Reply:
No worries! Feel free to ping me if you need me for anything.